src/components/button/button.stories.mdx

```diff
@@ -414,6 +414,72 @@ Passing in the `href` prop will render an anchor with `role="button"` and the Bu
   </Story>
 </Preview>
 
+### Icon Only Button
+
+Buttons can be rendered with just an icon. To achieve the correct styling for the icon placement, please pass `"center"` to the `iconPosition` prop. This ensures that the icon is centered within the button.
```

Comment:
Yeah I think we can get rid of this center stuff so the user doesn't have to do it themselves

Reply:
✅